### PR TITLE
i18n implementation and Chinese (Traditional/Simplified) translation

### PR DESCRIPTION
**Pull request by urakagi**
This pull request includes `tap-i18n` framework implementation and Chinese translation.
Please note that opposite to the guideline, language is auto-selected by browser's `navigator.language` in this PR, so users cannot set their language manually, and `Accept-Language` header is ignored. Also, `capnp` related texts are not touched. This will be left for further work.

### CODE REVIEW

File: shell/client/shell-client.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ import getBuildInfo from "/imports/client/build-info.js";
 import SandstormAccountSettingsUi from "/imports/client/accounts/account-settings-ui.js";
 import { isStandalone } from "/imports/client/standalone.js";
 
-// Subscribe to basic grain information first and foremost, since
+//ders Subscribe to basic grain information first and foremost, since
```
Comment (kentonv) [inline shell/client/shell-client.js]: typo

File: shell/client/shell-client.js [hunk 2]
```diff
@@ -31,6 +31,25 @@ globalSubs = [
...
 
+getUserLanguage = function () {
+  return navigator.language;
+};
+
+if (Meteor.isClient) {
+  Meteor.startup(function () {
+    Session.set("showLoadingIndicator", true);
+
+       TAPi18n.setLanguage(getUserLanguage())
```
Comment (kentonv) [inline shell/client/shell-client.js]: Indentation is off.

File: shell/client/shell-client.js [hunk 3]
```diff
@@ -31,6 +31,25 @@ globalSubs = [
...
+         .done(function () {
+            Session.set("showLoadingIndicator", false);
+         })
+         .fail(function (error_message) {
```
Comment (kentonv) [inline shell/client/shell-client.js]: Variable names should be camelCase, not under_scores.

File: shell/i18n/sync-langs.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```
Comment (kentonv) [inline shell/i18n/sync-langs.py]: Please add some comments to this file explaining what it does.

**Comment by urakagi** [comment]
I didn't meet any complicated string replacing for Chinese. Some strings can be fixed for more nature Chinese, but in fact we are used to read "translated text not so nature".

**Comment by kentonv** [comment]
@urakagi Can you add a commit addressing the comments I made?

**Comment by kentonv** [comment]
Also @urakagi and @caasi, can I ask you both to sign the Contributor License Agreement?

https://sandstorm.io/cla/

Thanks.

**Comment by urakagi** [comment]
@kentonv Okay, I have committed the fixes. I need to ask @audreyt about the CLA, please wait a moment.

**Comment by audreyt** [comment]
Green light 🥗  on the CLA, please use the individual version.

**Comment by urakagi** [comment]
I have signed the CLA. BTW, what's the conflict about shell/.meteor/versions? I think I didn't modify that?

**Comment by caasi** [comment]
@kentonv I just signed the individual CLA with my email address: caasi.igd {at} gmail.com.

**Comment by kentonv** [comment]
This has been released!

File: shell/packages/sandstorm-ui-topbar/topbar.html [hunk 5]
```diff
@@ -60,13 +61,13 @@
       {{#with accountExpires}}
       <li class="navitem-demo-notice">
         <div class="demo-notice">
-          <p>Start collaborating with others by using Sandstorm on Oasis:
-            <button class="button-primary sign-in">Create free account »</button>
-            <span class="subtext">Your demo data will be transferred.</span>
+          <p>{{_ (con txt "oasis")}}
+            <button class="button primary sign-in">{{_ (con txt "createAccount")}}</button>
```
Comment (kentonv) [inline shell/packages/sandstorm-ui-topbar/topbar.html]: Whoops, you changed the CSS class here... I'll fix it.

**Pull request merged**